linetools/isgm/abssightline.py

```diff
@@ -126,27 +120,24 @@ def __init__(self, radec, sl_type=None, em_type=None, comment=None, name=None):
 
         # Name
         if name is None:
-            self.name = 'J{:s}{:s}'.format(
-                    self.coord.ra.to_string(unit=u.hour,sep='',pad=True),
-                    self.coord.dec.to_string(sep='',pad=True,alwayssign=True))
+            self.name = ltu.name_from_coord(self.coord)
         else:
-            self.name=name
+            self.name = name
 
         # Others
         self.em_type = em_type
         self.sl_type = sl_type
+        self._abssystems = None
```

Comment:
add comment like: potentially we may want to save AbsSystems in this class but not implemented for now.